src/Card.js

```diff
@@ -411,7 +405,7 @@ Card.makeConfig = (config = {}) => {
     transform: Card.transform
   };
 
-  return _.assign({}, defaultConfig, config);
+  return Object.assign({}, defaultConfig, config);
```

Comment:
Might need a polyfill for this (I had one inline in AyUser where I needed it: https://github.com/AyogoHealth/ayogo-client-common/blob/master/src/user/user.ts#L50)